### PR TITLE
Fix/vanta docs

### PR DESCRIPTION
**Pull request by souredoutlook**
This resolves an error where customers would attempt to view the Vanta documentation from the Sentry web-app and an encounter a not-found error.

### Legal Boilerplate

Look, I get it. The entity doing business as "Sentry" was incorporated in the State of Delaware in 2015 as Functional Software, Inc. and is gonna need some rights from me in order to utilize my contributions in this here PR. So here's the deal: I retain all rights, title and interest in and to my contributions, and by keeping this boilerplate intact I confirm that Sentry can use, modify, copy, and redistribute my contributions, under Sentry's choice of terms.

### CODE REVIEW

**Comment by souredoutlook** [comment]
Hey @ceorourke can you have a peek at this deploy and let me know if it looks okay.

I linked to the Vanta docs originally but saw that it broke the product link

File: src/docs/product/integrations/compliance/vanta/index.mdx [hunk 1]
```diff
@@ -0,0 +1,24 @@
+---
+title: Vanta
+sidebar_order: 1
+redirect_from:
+  - /product/integrations/vanta/
+description: "Learn about Sentry's Vanta integration and how it to helps automate security compliance. ."
```
Comment (lizokm) [inline src/docs/product/integrations/compliance/vanta/index.mdx:6]:
```suggestion
description: "Learn about Sentry's Vanta integration and how it helps automate security compliance."
```

File: src/docs/product/integrations/compliance/vanta/index.mdx [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+---
+
+The Vanta integration for Sentry grants Vanta read-only access to help verify that Sentry organization members are associated with employees in your organization and helps automate security compliance.
+
+
+This integration is maintained and supported by the company that created it. For more details, check out our [Integration Platform documentation](/product/integrations/integration-platform/).
+
```
Comment (lizokm) [inline src/docs/product/integrations/compliance/vanta/index.mdx:13]:
```suggestion
Sentry's Vanta integration grants Vanta read-only access to verify that Sentry organization members are associated with employees in your organization, which automates security compliance. This integration is maintained and supported by the company that created it. For more details, check out our [Integration Platform documentation](/product/integrations/integration-platform/).

```

File: src/docs/product/integrations/compliance/vanta/index.mdx [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+<Note>
+
+Sentry owner, manager, or admin permissions are required to install this integration.
+
+</Note>
+
+1. Navigate to **Settings > Integrations > Vanta**
+
+2. Follow the full [Vanta installation instructions](https://help.vanta.com/hc/en-us/articles/13045851267092).
```
Comment (lizokm) [inline src/docs/product/integrations/compliance/vanta/index.mdx:24]:
```suggestion

To set up the Vanta integration:

1. Navigate to **Settings > Integrations > Vanta**

2. Follow the full [Vanta installation instructions](https://help.vanta.com/hc/en-us/articles/13045851267092).
```

**Review by lizokm — APPROVED** [review]
When I looked at this in the preview, it looked like the entire side nav was broken, but I'm not sure if that's just in the preview....

Other than that, made a few tweaks, but overall this LGTM

File: src/docs/product/integrations/index.mdx [hunk 4]
```diff
@@ -23,7 +23,7 @@ For more details, see the [full Integration Platform documentation](/product/int
 
 | Integration                                                      | API |
 | ---------------------------------------------------------------- | --- |
-| [Vanta](https://help.vanta.com/hc/en-us/articles/13045851267092) | X   |
+| [Vanta](/product/integrations/compliance/vanta/)                 | X   |
```
Comment (ceorourke) [inline src/docs/product/integrations/index.mdx:26]: Do you mind moving this section much lower on the page, maybe just above Cloud Monitoring?

**Comment by souredoutlook** [comment]
@ceorourke is this link broken? Wondering if I messed up the redirect value? https://sentry-docs-qfpbym3ej.sentry.dev/product/integrations/compliance/

**Comment by ceorourke** [comment]
> https://sentry-docs-qfpbym3ej.sentry.dev/product/integrations/compliance/



> @ceorourke is this link broken? Wondering if I messed up the redirect value? https://sentry-docs-qfpbym3ej.sentry.dev/product/integrations/compliance/

It works for me, what do you see?

**Comment by souredoutlook** [comment]
> > https://sentry-docs-qfpbym3ej.sentry.dev/product/integrations/compliance/
> 
> > @ceorourke is this link broken? Wondering if I messed up the redirect value? https://sentry-docs-qfpbym3ej.sentry.dev/product/integrations/compliance/
> 
> It works for me, what do you see?

File: src/docs/product/integrations/compliance/index.mdx [hunk 5]
```diff
@@ -0,0 +1,11 @@
+---
+title: Compliance
+sidebar_order: 95
+description: "Learn more about Sentry's compliance integrations."
+redirect_from:
```
Comment (ceorourke) [inline src/docs/product/integrations/compliance/index.mdx:5]: Try removing the `redirect_from` here, it's not needed and I think causing the issues you were talking about.

Comment (souredoutlook) [reply src/docs/product/integrations/compliance/index.mdx]: Thanks! Appreciate you taking a look!

**Review by ceorourke — APPROVED** [review]

**Pull request merged**